DrewBradley/rancid-tomatillos
# Feature/error handling

### What does this PR do?
- This PR displays error messages on initial page load as well as when you try to view an individual movie and it's details.
- This was tested by corrupting the file path in the API fetch request. We will test for 500 responses later.

### What does it fix?
- No bugs to fix.

### Is this a feature or a fix?
- This is a feature, providing an error message to the user.

### Where should the reviewer start?
- Clone this repo down to your machine. Run npm install then run npm start. 

### How should this be tested?
- While in MainDisplay.js, try messing up the file path for each one of the fetch calls respectively. Start with the get request for all movies and verify that an error message pops up. Then fix that get request and modify the get request for the individual movie call, try clicking on an individual movie and you should see an error message as well.

@letakeane
@sertmer
